src/templates/Article/index.js

```diff
@@ -123,49 +136,194 @@ const figure = {
   }),
   editorModule: 'figure',
   editorOptions: {
-    pixelNote: 'Auflösung: min. 1200x, für E2E min. 2000x (proportionaler Schnitt)',
-    insertButtonText: 'Bild',
+    pixelNote:
+      'Auflösung: min. 1200x, für E2E min. 2000x (proportionaler Schnitt)',
     sizes: [
       {
         label: 'Edge to Edge',
-        props: {size: undefined},
-        parent: {kinds: ['document', 'block'], types: ['CENTER']},
+        props: { size: undefined },
+        parent: {
+          kinds: ['document', 'block'],
+          types: ['CENTER']
+        },
         unwrap: true
       },
       {
         label: 'Gross',
-        props: {size: 'breakout'},
-        parent: {kinds: ['document', 'block'], types: ['CENTER']},
+        props: { size: 'breakout' },
+        parent: {
+          kinds: ['document', 'block'],
+          types: ['CENTER']
+        },
         wrap: 'CENTER'
       },
       {
         label: 'Normal',
-        props: {size: undefined},
-        parent: {kinds: ['document', 'block'], types: ['CENTER']},
+        props: { size: undefined },
+        parent: {
+          kinds: ['document', 'block'],
+          types: ['CENTER']
+        },
         wrap: 'CENTER'
       }
     ]
   },
+  rules: [
+    figureImage,
+    figureCaption
+  ]
+}
+
+const floatFigureCaption = {
+  ...figureCaption,
+  editorOptions: {
+    ...figureCaption.editorOptions,
+    type: 'FLOATFIGURECAPTION',
+    afterType: 'PARAGRAPH',
+    insertAfterType: 'CENTER'
+  },
   rules: [
     {
-      matchMdast: matchImageParagraph,
-      component: FigureImage,
-      props: (node, index, parent, { ancestors }) => {
-        const src = extractImage(node)
-        const displayWidth = getDisplayWidth(ancestors)
+      ...figureByLine,
+      editorOptions: {
+        ...figureByLine.editorOptions,
+        type: 'FLOATEMPHASIS'
+      }
+    },
+    link,
+    ...globalInlines
+  ]
+}
 
-        return {
-          ...FigureImage.utils.getResizedSrcs(
-            src,
-            displayWidth
-          ),
-          alt: node.children[0].alt
+const floatFigure = {
```

Comment:
would prefer the naming `centerFigure`—was trigging to figure out why the figure floats ;)